### ISSUE TITLE
convert tiny-yolo from darknet to caffe without bn

### ISSUE THREAD

**Issue by leexp0304** [marvis/pytorch-caffe-darknet-convert#58]
what means without bn?

**Comment by hammadullah125** [comment]
Batch Normalization
